airflow/ui/src/pages/Dag/Runs/Runs.tsx

```diff
@@ -89,7 +89,10 @@ const columns: Array<ColumnDef<DAGRunResponse>> = [
   },
   {
     cell: ({ row: { original } }) =>
-      `${dayjs.duration(dayjs(original.end_date).diff(original.start_date)).asSeconds().toFixed(2)}s`,
+      `${dayjs
+        .duration(dayjs(original.end_date ?? dayjs()).diff(original.start_date))
+        .asSeconds()
+        .toFixed(2)}s`,
```

Comment:
Let's turn this into a reusable `getDuration(startDate, endDate)` util.

Also, we can just do `end_date ?? undefined`. The problem is that dayjs() can't properly handle `null`

Reply:
Thanks, makes sense. The legacy utils has `getDuration` based on moment. Refactored all changes to use `getDuration` as suggested.